public/calendar_reboot.js

```diff
@@ -69,6 +97,24 @@ function CheckIt() {
 
 };
 
+
+CheckIt.prototype.displayLoadingWheel = function(elementId) {
+    // Displays the loading wheel.
+    
+    // Parameters: elementId 
+           // A string pertaining to the id of the element
+           // where you want to place the loading wheel.
+    
+    // Needs a plugin to use jQuery.
+    var target = document.getElementById(elementId);
```

Comment:
You can still user jQuery here and just grab the element out of it before you pass it to the spinner.
I think it's `this.spinner.spin($(selector).get());`.